packages/docusaurus-theme-classic/src/theme/CodeBlock/index.js

```diff
@@ -40,6 +40,7 @@ export default ({children, className: languageClassName, metastring}) => {
 
   const target = useRef(null);
   const button = useRef(null);
+  const fileName = languageClassName.split(':')[1];
```

Comment:
Can we make this feature more general? It seems to me that we need to look at this as a code title, not as just a code filename. For example, I want to add some text that has spaces in it, will it work?

> 
> \`\`\`js:"My Code title" {2}
> console.log('123');
> \`\`\`
>

Reply:
Agree! And currently your example not working 😢

Reply:
Could you add such an opportunity?

Reply:
sure!